src/main/java/com/sprint/mission/discodeit/async/binary_content/BinaryContentStorageAsyncService.java

```diff
@@ -0,0 +1,76 @@
+package com.sprint.mission.discodeit.async.binary_content;
+
+
+import com.sprint.mission.discodeit.async.AsyncTaskFailure;
+import com.sprint.mission.discodeit.async.AsyncTaskFailureRepository;
+import com.sprint.mission.discodeit.entity.NotificationType;
+import com.sprint.mission.discodeit.entity.User;
+import com.sprint.mission.discodeit.event.event_entity.NotificationEvent;
+import com.sprint.mission.discodeit.exception.file.FileException;
+import com.sprint.mission.discodeit.storage.BinaryContentStorage;
+import java.util.List;
+import java.util.UUID;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.slf4j.MDC;
+import org.springframework.context.ApplicationEventPublisher;
+import org.springframework.retry.annotation.Backoff;
+import org.springframework.retry.annotation.Recover;
+import org.springframework.retry.annotation.Retryable;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class BinaryContentStorageAsyncService {
+
+  private final BinaryContentStorage binaryContentStorage;
+  private final AsyncTaskFailureRepository asyncTaskFailureRepository;
+  private final ApplicationEventPublisher eventPublisher;
+
+  @Retryable(
+      backoff = @Backoff(delay = 1000),
+      retryFor = FileException.class
+  )
+  public boolean uploadFile(User user, UUID id, byte[] bytes) {
+    try {
+
+      binaryContentStorage.put(id, bytes);
+      return true;
+//      return CompletableFuture.completedFuture(true);
+    } catch (FileException e) {
+      // logging
+      throw e;
+    }
+  }
+
+  @Recover
+  @Transactional(propagation = Propagation.REQUIRES_NEW)
+  public boolean recover(FileException e, User user, UUID id, byte[] bytes) {
+
+    log.info("requestId in recover = {}", MDC.get("requestId"));
+    AsyncTaskFailure failure = new AsyncTaskFailure();
+    failure.setFields(
+        "BinaryContentStorage#put",
+        MDC.get("requestId"),
```

Comment:
```suggestion
        MDC.get("requestId"),
```
requestId 와 같이 자주 사용될 수 있는 상수는 따로 선언해두면 좋습니다!